client/components/thank-you/index.tsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * External dependencies
+ */
+import { TranslateResult, useTranslate } from 'i18n-calypso';
```

Comment:
Minor note: I think we can split this up so we're only importing the type for `TranslateResult` - I believe that means we only use the import during compilation and not at runtime.
```suggestion
import type { TranslateResult } from 'i18n-calypso';
import { useTranslate } from 'i18n-calypso';
```